src/middlewared/middlewared/plugins/config.py

```diff
@@ -105,7 +106,10 @@ def upload(self, job):
             is_tar = tarfile.is_tarfile(stf.name)
             self.upload_impl(stf.name, is_tar_file=is_tar)
 
-        self.middleware.run_coroutine(self.middleware.call('system.reboot', {'delay': 10}), wait=False)
+        self.middleware.run_coroutine(
+            self.middleware.call('system.reboot', 'Configuration upload', {'delay': 10}, app=app),
```

Comment:
The `Configuration upload` string is typed many times and is prone to typos. This is why I asked for a `RebootReasons` enum in a previous PR. Let's put all these statically typed strings in that class and re-use them everywhere.

Reply:
@yocalebo it's a different class of strings. The `RebootReasons` class we added is why the system must be rebooted. This is the string that will be logged to the audit log when the system is rebooted.

Do you want another centralized storage for all the reasons we reboot the system ourselves, or will a per-file string constant(s) suffice? I see no reason to centralize this, the reboot/shutdown reason supplied by user can be arbitrary.

Reply:
Per-file string constants are fine for this. We just need to focus on the details here since this will show up in audit logs and we need to be consistent.